### PR TITLE
Update repo with '22 program changes

### PR DESCRIPTION
**Pull request by jsoares**
* Updates repo readme to incorporate program changes (and RFP-X)
* Adds publication policy
* Adds empty FAQ (@ebryson2 please complete!)
* Adds list of other grant programs

### CODE REVIEW

**Comment by jsoares** [comment]
@ebryson2 Please add your existing FAQs to the new file. This needn't be final: we'll start with what we have and update as we go.

@asilinphoto @ebryson2 Please review. Note that the new Apply program has been updated with links that are broken until this merges, while the existing/public Apply pages still haven't been updated. This is a chicken-and-egg problem (more of a circular dependency) so something will be broken for some time while we merge. I'd love to merge this today (Thursday) to then update Apply and launch the new programs on Friday.

**Comment by asilinphoto** [comment]
Awesome! Holding off on approving until FAQ info has been added. @jsoares here are some notes/questions from the remaining sections:

Publications.md - lines 8 - 11 - Can we indent these lines to match the style in Readme.md? Also, do we want these all starting with capitalized or uncapitalized words?
      line 5 uses e.g., line 19 uses i.e.. Which is preferred?
      
Readme.md, line 1 - Since the title is now referencing grants, do we want to update the link to the /research-grants repo? Or is this being left as is so there's more information about PL Research?
      line 94 - add colon after "for"

**Comment by jsoares** [comment]
> Publications.md - lines 8 - 11 - Can we indent these lines to match the style in Readme.md? Also, do we want these all starting with capitalized or uncapitalized words? 

Lines now indented but note that this renders the same in GitHub. Also capitalised each line.

> line 5 uses e.g., line 19 uses i.e.. Which is preferred?

Not sure what you mean; they both seem to be used correctly and I don't see any , or . after 🤔 

> Readme.md, line 1 - Since the title is now referencing grants, do we want to update the link to the /research-grants repo? Or is this being left as is so there's more information about PL Research? 

Also not sure here. The repo is research-grants already? 

> line 94 - add colon after "for"

done

**Comment by asilinphoto** [comment]
@jsoares thanks! I wasn't aware the indents rendered the same either way, so that's helpful to know.
 - Readme.md line 1 - scratch that, you're good!
 - i.e. vs e.g. - I'm thinking we are probably reading the sentences differently to ourselves. When I read both sentences, the information in parenthesis seemed they could both either be used to clarify the first part of the sentences _or_ be used as an example. Essentially, I read them both with the same tone so thought both sentences should either use i.e. or e.g. instead of one of each 🤷‍♀️

**Comment by jsoares** [comment]
I'd say [line 5] community events are just one example of non-scientific events (so eg), whereas [line 19], articles being available online and immediately is the [definition of Gold OA](https://authorservices.taylorandfrancis.com/publishing-open-access/oa-green-gold/) (so ie).

**Comment by ebryson2** [comment]
Line 32: Can we change MSc to master's? To be more inclusive of standards in different countries

File: README.md
Comment (jsoares) [inline README.md]:
```suggestion
The Doctoral Fellowship is intended to support the development of an early-stage researcher conducting research in one of PL's fields of interest. This award is available to doctoral candidates who have completed their Master's degree or have otherwise satisfied the requirements for candidacy.
```

Comment (jsoares) [reply README.md]: @protocol/research-acceleration ^^ suggestions really are your friends

I think that's a good idea, but then I think we should do the same for PhD

File: README.md
Comment (jsoares) [inline README.md]:
```suggestion
    * Doctoral Fellowships
```

File: README.md
Comment (jsoares) [inline README.md]:
```suggestion
The **PL Research Doctoral Fellowship** consists of a fixed award of **$60,000**, intended to cover tuition and subsistence costs for a period of one year. Doctoral Fellowships can be renewed for a second year, by mutual agreement between PL and the grantee.
```

File: README.md
Comment (jsoares) [inline README.md]:
```suggestion
This application may be submitted by the doctoral candidate, their PI, or an administrative staff member from the host institution. The submitter will serve as the primary point of contact for all communications regarding this application.
```

File: FAQ.md
Comment (jsoares) [inline FAQ.md]:
```suggestion
A: Each open grant and RFP has its own set of requirements; these are clearly stated in the application portal. Most will require a detailed proposal and a CV. Doctoral and postdoctoral fellowships also require a letter of recommendation, which can be submitted directly by the applicant or emailed to research-acceleration@protocol.ai by the recommender. 
```

File: FAQ.md
Comment (jsoares) [inline FAQ.md]:
```suggestion
A: Doctoral and postdoctoral fellowship proposals may be submitted by an advisor or by administrative staff at the candidate's institution. Please note that all requirements must still be met, including a detailed research proposal and letter of recommendation.
```

**Comment by jsoares** [comment]
This is already late -- we promised delivery this week. Please review when possible so we can go ahead and merge.

File: README.md
Comment (ebryson2) [inline README.md]: Is this preferred over the research acceleration address?

**Review by asilinphoto — APPROVED** [review]
FAQs @ebryson2 rereading this, can we adjust line 4 from "these are clearly stated in the application portal" to "these can be found in the [relevant programme description](https://research.protocol.ai/outreach/)" ?

You already pointed out the research-acceleration@ vs research@ email address for outreach.

Approving because previous adjustments read well to me!

**Comment by ebryson2** [comment]
> FAQs @ebryson2 rereading this, can we adjust line 4 from "these are clearly stated in the application portal" to "these can be found in the [relevant programme description](https://research.protocol.ai/outreach/)" ?

@asilinphoto Los dos? There is more detail in the portal

File: README.md
Comment (jsoares) [reply README.md]: We generally avoid disclosing the internal team groups. That said, research-grants@ could be a better option?

File: FAQ.md
Comment (jsoares) [inline FAQ.md]:
```suggestion
Each open grant and RFP has its own set of requirements; these are clearly stated in the application portal. Most will require a detailed proposal and a CV. Doctoral and postdoctoral fellowships also require a letter of recommendation, which must be submitted with the application. 
```

File: FAQ.md
Comment (jsoares) [inline FAQ.md]:
```suggestion
A: Please reach out to research-grants@protocol.ai.
```

**Comment by asilinphoto** [comment]
@ebryson2 What does 'los dos' mean in this context? 😆

To your point - yes, though it's possible folks want to know what's needed with the application prior to starting one or knowing that they'll end up creating a portal as part of the process. I figured it would be good to approach without the assumption that someone is aware they will (or do) have their own portal. It also reads a bit harsh out of context of our conversation

**Comment by ebryson2** [comment]
@asilinphoto We could do both (i.e. the relevant programme description can point you in the right direction, but know that there is more detail in the portal to guide you, and hey, reach out to us if there are still questions as you work through the application). I'm not detecting the Harshness, but I'll trust your reading of it

**Comment by asilinphoto** [comment]
@ebryson2 I like that! Yes, let's provide both in the answer

File: README.md
Comment (jsoares) [reply README.md]:
```suggestion
If you have any questions, please email us at **research-grants@protocol.ai**.
```

File: FAQ.md
Comment (jsoares) [reply FAQ.md]: We dropped the option of emailing later as part of the streamlining

Comment (jsoares) [reply FAQ.md]: Also, very few things are now stated in the portal; it just redirects people to GitHub for further information.

**Comment by jsoares** [comment]
> We could do both (i.e. the relevant programme description can point you in the right direction, but know that there is more detail in the portal to guide you, and hey, reach out to us if there are still questions as you work through the application). I'm not detecting the Harshness, but I'll trust your reading of it

I'm not sure I know what you're talking about, but all the information is now in the github repo. The portal only has a brief description of each type of grant and redirects to github. So I don't see much point in sending people to the portal. We should send people to the readme and perhaps mention that the application process is guided?

File: FAQ.md
Comment (ebryson2) [reply FAQ.md]: > We dropped the option of emailing later as part of the streamlining

It wasn't a big lift for me to add these to proposals in Apply. Even though applicants will be able to see them whether we upload them ourselves or they do, I'm a little concerned that some advisors won't want to do this. More than half in the last round chose to email directly.

> Also, very few things are now stated in the portal; it just redirects people to GitHub for further information.

Is it possible we're talking about different things? This question is usually about the minutiae of proposals, which afaik is best explained in the task instructions in Apply.

Comment (jsoares) [reply FAQ.md]: > It wasn't a big lift for me to add these to proposals in Apply. Even though applicants will be able to see them whether we upload them ourselves or they do, I'm a little concerned that some advisors won't want to do this. More than half in the last round chose to email directly.

It wasn't a big lift in terms of actions, but it also meant applications might be stalled for weeks, be mistracked, etc. Our SLAs for this round were pretty bad, with applications remaining in screening long over 48h (after the deadline). This really shouldn't happen. 

It should come as no surprise that I'd make everything public if it were my call. There is no acceptable reason why an advisor would not want to disclose a letter to their employee, unless they're getting ready to waste everyone's time. 

> This question is usually about the minutiae of proposals, which afaik is best explained in the task instructions in Apply.

Understood, but we need to write that clearly. Right now we're redirecting people to a page that redirects them right back.

**Pull request merged**
